ecommerce/api.py

```diff
@@ -204,3 +209,38 @@ def get_new_order_by_reference_number(reference_number):
         return Order.objects.get(id=order_id, status=Order.CREATED)
     except Order.DoesNotExist:
         raise EcommerceException("Order {} is expected to have status 'created'".format(order_id))
+
+
+def enroll_user(order):
+    """
+    Enroll user after they made a successful purchase.
+
+    Args:
+        order (Order): An order to be fulfilled
+
+    Returns:
+         None
+    """
+    user_social = order.user.social_auth.get(provider=EdxOrgOAuth2.name)
+    enrollments_client = EdxApi(user_social.extra_data, settings.EDXORG_BASE_URL).enrollments
+
+    exceptions = []
+    enrollments = {}
+    for line in order.line_set.all():
+        course_key = line.course_key
+        try:
+            enrollments[course_key] = enrollments_client.create_audit_student_enrollment(course_key)
+        except Exception as ex:  # pylint: disable=broad-except
```

Comment:
You're right, I'd like to leave it in case the function implementation changes in the future and they forget to update this part